pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -242,9 +242,10 @@ func (c *Operation) QueryConnections(rw http.ResponseWriter, req *http.Request)
 //        200: queryConnectionResponse
 func (c *Operation) QueryConnectionByID(rw http.ResponseWriter, req *http.Request) {
 	params := mux.Vars(req)
-	logger.Debugf("Querying connection invitation for id [%s]", params["id"])
+	logger.Infof("Querying connection invitation for id [%s]", params["id"])
```

Comment:
Looks like log levels need to be cleaned up in this PR.

Reply:
done